text/0001-cdk-update.md

```diff
@@ -0,0 +1,293 @@
+---
+rfc pr: [#335](https://github.com/aws/aws-cdk-rfcs/pull/335)
+tracking issue: https://github.com/aws/aws-cdk-rfcs/issues/1
+---
+
+# CDK Update
```

Comment:
We call this feature "CDK Watch" (see tracking issue) to signal to users that this feature _must not_ be used in production context

Reply:
I am not 100% on board with that change of name. "watch" implies a persistent process, whereas this _may_ do that, but may also be a one-shot.

Reply:
We also discussed the possibility of `cdk accelerate` to align with similar features planned in the SAM Accelerate project.

Reply:
I am okay with @skinny85's suggestion for the title.

We will make a decision about the "marketing name" later.